python/pythonlab/neighborhood/neighborhood/painter.py

```diff
@@ -22,13 +22,15 @@ def turn_left(self):
     """
     Turn the painter one compass direction left (i.e. North -> West).
     """
-    print(f'{PAINTER_MESSAGE_PREFIX} {NeighborhoodSignalKey.TURN_LEFT.value}')
+    signal_message = NeighborhoodSignalMessage(SignalMessageType.PAINTER, NeighborhoodSignalKey.TURN_LEFT.value)
+    return signal_message.get_formatted_message()
```

Comment:
We want to print these, not return them. We are going to intercept the system out messages on the frontend (we'll intercept ones that start with the signal key) and use them to update the neighborhood visualization.

Reply:
Oh interesting. Will update!